functions-and-scope.js

```diff
@@ -65,6 +96,17 @@ console.log(List);
 // averageGrade([6, 4, 5]) geeft 5
 // averageGrade([8, 9, 4, 6, 10]) geeft 7.4
 
+function averageGrade(grades) {
+    let sum = 0;
+    const numberOfGrades = grades.length;
+    for (let i=0; i<grades.length; i++) {
+        sum += grades[i]
+    }
+    const gemiddelde3=sum/numberOfGrades;
```

Comment:
Maak gebruik van spaties voor de leesbaarheid, probeer ook alles in het Engels te doen. Engels is de standaard taal.